pkg/sdk/symbols/info/info_test.go

```diff
@@ -102,3 +104,49 @@ func TestInfo(t *testing.T) {
 		t.Errorf("(extractEventSources) expected %s, but found %s", testStr, resStr)
 	}
 }
+
+func TestSemver(t *testing.T) {
+	t.Run("success_check", func(t *testing.T) {
+		version := "2.0.0"
+		panicFunc := func() {
+			SetRequiredAPIVersion(version)
+		}
+		assert.NotPanics(t, panicFunc)
+	})
+
+	t.Run("invalid_version", func(t *testing.T) {
+		version := "invalid"
```

Comment:
A good test case could also be `version := ""` since this is the default value that `SetRequiredAPIVersion` API will receive when the plugin creator doesn't specify an explicit `RequiredAPIVersion`

Reply:
Sure will do !!